src/script/signingprovider.cpp

```diff
@@ -368,6 +368,7 @@ TaprootBuilder& TaprootBuilder::Add(int depth, Span<const unsigned char> script,
     /* Construct NodeInfo object with leaf hash and (if track is true) also leaf information. */
     NodeInfo node;
     node.hash = ComputeTapleafHash(leaf_version, script);
+    // NOLINTNEXTLINE(modernize-use-emplace)
```

Comment:
Can you explain why in the source comment? (at least outside of test and fuzz code)